decompiler/config/all-types.gc

```diff
@@ -22037,8 +22036,6 @@
   :size-assert         #x7c
   :heap-base           #x100
   :flag-assert         #xe0100007c
-  (:states
-    part-tester-idle)
```

Comment:
same here

Reply:
it's not useful, the state is defined before use